lib/ramble/ramble/cmd/results.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import llnl.util.tty as tty
+
+from ramble.config import ConfigError
+import ramble.experimental.uploader
+
+
+description = "take actions on experiment results"
+section = "workspaces"
+level = "short"
+
+subcommands = [
+    'file_upload',
+]
+
+#: Dictionary mapping subcommand names and aliases to functions
+subcommand_functions = {}
+
+
+def results_file_upload_setup_parser(subparser):
+    subparser.add_argument(
+        'filename', metavar='filename',
+        help='path of file to upload')
+
+
+def results_file_upload(args):
```

Comment:
This logic looks like a replication of the upload logic in workspace.py. It might be nice to DRY it out and push it into the uploader class instead of having it exist in all of the individual classes. WDYT?

Reply:
Completed this